### ISSUE TITLE
[BUG] Can't sort by non-default Torrent Detail Columns in Expanded View 

### ISSUE THREAD

**Issue by NoLooseEnds** [jfurrow/flood#395]
If I i.e tick off "Creation Date" in the Interface settings in Expanded View, it does not show up in the list where you sort stuff. 

Settings:
<img width="628" alt="flood" src="https://user-images.githubusercontent.com/4103531/27772847-3b74edc6-5f62-11e7-8e9a-a5a58e2611d1.png">

Sorting list, missing "Creation Date":
<img width="243" alt="flood" src="https://user-images.githubusercontent.com/4103531/27772851-5220dc4c-5f62-11e7-8afc-18a42d9676de.png">

**Comment by jfurrow** [comment]
Yeah, this is true... the dropdown should be dynamically populated.
